frontend/src/boot/env-parser.js

```diff
@@ -1,5 +1,10 @@
 const DotEnv = require("dotenv");
+var shell = require("shelljs");
```

Comment:
How does this dependency affect the app bundle size? NPM says it's 211Kb unpacked:

https://www.npmjs.com/package/shelljs

Is there a smaller, single-purpose library that can be used or perhaps a way to do this with the packages already included in the app such as Axiom? Here is an example of a smaller, single-purpose library:

https://www.npmjs.com/package/is-port-reachable

Reply:
Removed the dependency altogether.